terraform/modules/quorum-cluster-region/quorum.tf

```diff
@@ -496,39 +496,69 @@ data "aws_ami" "quorum" {
 # ---------------------------------------------------------------------------------------------------------------------
 # OUTPUT INSTANCES
 # ---------------------------------------------------------------------------------------------------------------------
-data "aws_instance" "quorum_maker_node" {
-  count = "${aws_autoscaling_group.quorum_maker.count}"
+data "aws_instances" "quorum_maker_dns" {
```

Comment:
Why introduce `aws_instances` here? What do you get out of it that you can't get out of individual `aws_instance` data sources?  I don't mind the idea of replacing multiple `aws_instance` data sources with a single `aws_instance`, but I strongly prefer not to include both.

This comment applies to other node types as well

Reply:
aws_instances is introduced because your final list of output (maker_node, observer_node, validator_node) is empty after first runs.